custom-recipes/googlesheets-append/recipe.py

```diff
@@ -20,14 +24,16 @@
 
 # Get configuration
 config = get_recipe_config()
+logger.info("config parameters: {}".format(logger.filter_secrets(config)))
 credentials = config.get("credentials")
 doc_id = config.get("doc_id")
 tab_id = config.get("tab_id")
 insert_format = config.get("insert_format")
+session = GoogleSheetsSession(credentials)
 
 
 # Load worksheet
-ws = get_spreadsheet(credentials, doc_id, tab_id)
+worksheet = session.get_spreadsheet(doc_id, tab_id)
```

Comment:
Might be good to note the first spreadsheet is used when tab_id is not provided.
I think  if you miss this in the UI it is flagged as red but things still work? Not sure if that is intended.

Reply:
I tested again - this is what happens. Maybe we should stop it - seems a bit risky/random

Reply:
👍 fixed in [938cffa](https://github.com/dataiku/dss-plugin-googlesheets/pull/1/commits/938cffa76498996ca84499fc2610b54177f86db0), we now raise if no tab id is provided to the recipe

Reply:
Great - I have tested that, all good